unime/src/routes/(app)/me/settings/+page.svelte

```diff
@@ -31,36 +22,44 @@
     />
   </div>
   <!-- Account -->
-  <div class="flex flex-col space-y-[10px]">
+  <div class="flex flex-col gap-3">
     <p class="text-[14px]/[22px] font-medium text-slate-500 dark:text-slate-300">{$LL.SETTINGS.ACCOUNT()}</p>
-    <SettingsEntry
-      icon={UserCircleFillIcon}
-      title={$LL.SETTINGS.PROFILE.TITLE()}
-      on:click={() => goto('/me/settings/profile')}
-    />
-    <SettingsEntry icon={GearFillIcon} title={$LL.SETTINGS.APP.TITLE()} on:click={() => goto('/me/settings/app')} />
-    <SettingsEntry
-      icon={VaultFillIcon}
-      title={$LL.SETTINGS.BACKUP_RECOVERY.TITLE()}
-      on:click={() => goto('/me/settings/backup')}
-      disabled
-    />
-    <SettingsEntry icon={SignOutFillIcon} title={$LL.SETTINGS.LOG_OUT.TITLE()} hasCaretRight={false} disabled />
+
+    <SettingsCaretLink href="/me/settings/profile">
+      {#snippet icon()}
+        <UserCircleFillIcon class="h-5 w-5 text-primary"></UserCircleFillIcon>
```

Comment:
All square elements that have the same height and width (e.g. `h-5 w-5`) can be expressed as `size-5`.